indico/MaKaC/webinterface/tpls/latex/report.tpl

```diff
@@ -30,8 +30,8 @@
     % for item in items:
         \newpage
         \fancyhead[L]{\small \rmfamily \color{gray} \truncateellipses{${conf.getTitle() | latex_escape}}{80pt} / ${title}}
-        \fancyhead[R]{\small \rmfamily \color{gray} \truncateellipses{${item.getTitle() | latex_escape}}{150pt}}
-        \addcontentsline{toc}{section}{${item.getTitle() | latex_escape}}
+        \fancyhead[R]{\small \rmfamily \color{gray} \truncateellipses{${item.title or item.getTitle() | latex_escape}}{150pt}}
+        \addcontentsline{toc}{section}{${item.title or item.getTitle() | latex_escape}}
```

Comment:
We are using the `report.tpl` in `AbstractsToPDF` and `item.getTitle()` is a fallback in case that we have to retrieve title from a legacy object.